packages/cli/src/interact.ts

```diff
@@ -30,19 +30,22 @@ export async function interact(ctx: InteractTaskArgs) {
 
   await printHeader(ctx);
 
+  let interacting = true;
   let pickedContract: string | null = null;
   let pickedFunction: string | null = null;
   let currentArgs: any[] | null = null;
   let txnValue: Wei = wei(0);
 
   // eslint-disable-next-line no-constant-condition
-  while (true) {
+  while (interacting) {
     if (!pickedContract) {
       pickedContract = await pickContract({
         contractNames: _.keys(ctx.contracts),
       });
 
-      if (!pickedContract) {
+      if (pickedContract == PROMPT_BACK_OPTION.title) {
+        interacting = false;
```

Comment:
IMHO, instead of adding an extra `interacting` variable, you could just use `break;` here to get out of the loop.